fixtures/library-file/src/styles.treat.ts

```diff
@@ -0,0 +1,9 @@
+// @ts-ignore
+import { style } from 'sku/treat';
```

Comment:
I don't think ts-ignore is needed anymore. If it was copy-pasted from another fixture, it's probably not needed there anymore either.

Reply:
Yes, the `library-file` fixture was copied from `library-build` with minimal changes.